x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -144,17 +144,18 @@ const getSavedObjectTypes = (
         ssl: { type: 'binary' },
         proxy_id: { type: 'keyword' },
         shipper: {
+          dynamic: false,
           properties: {
-            disk_queue_enabled: { type: 'boolean', index: false },
-            disk_queue_path: { type: 'keyword', index: false },
-            disk_queue_max_size: { type: 'integer', index: false },
-            disk_queue_encryption_enabled: { type: 'boolean', index: false },
-            disk_queue_compression_enabled: { type: 'boolean', index: false },
-            compression_level: { type: 'integer', index: false },
-            loadbalance: { type: 'boolean', index: false },
-            mem_queue_events: { type: 'integer', index: false },
-            queue_flush_timeout: { type: 'integer', index: false },
-            max_batch_bytes: { type: 'integer', index: false },
+            disk_queue_enabled: { type: 'boolean' },
+            disk_queue_path: { type: 'keyword' },
+            disk_queue_max_size: { type: 'integer' },
+            disk_queue_encryption_enabled: { type: 'boolean' },
+            disk_queue_compression_enabled: { type: 'boolean' },
+            compression_level: { type: 'integer' },
+            loadbalance: { type: 'boolean' },
+            mem_queue_events: { type: 'integer' },
+            queue_flush_timeout: { type: 'integer' },
+            max_batch_bytes: { type: 'integer' },
```

Comment:
If we had `index: false` before it means we're not querying any of these fields. Now that we have `dynamic: false` we should remove all of them. Unless we're querying or aggregating there should be no field in the mappings.

Reply:
I removed the properties as requested. @rudolf could you please review again?